docs/development_guide.md

```diff
@@ -152,3 +164,17 @@ To branch the stable version and let master be the new devel branch again:
   1. Run `bin/rake update_versions`, this will update all references to the new version.
   1. Run `bin/rake bundle`, this will update all the `Gemfile.lock` files
   1. Run `bin/rake webpack`, this will update the JavaScript bundle.
+
+
+Releasing new ***patch versions*** is quite easy, it's the same process whether it's a new version or a patch:
```

Comment:
> it's the same process whether it's a new version or a patch: 

if it's the same process should we explain it again?

Reply:
I left it here as this PR is still a draft and it was like this in the previous version of the documentation.
The implied branches change if releasing news versions of stable branches. I think that creating a "Releasing new x.y-stable versions" title but keeping the chapter (updated) will be good.